typesafe_config_base/README.md

```diff
@@ -0,0 +1,16 @@
+# wellcome/typesafe_config_base
+
+Base container image for typesafe services at Wellcome.
+
+## Usage
+
+This container is intended to be extended from like so:
+
+```
+FROM wellcome/typesafe_config_base:104
+
+ADD target/universal/stage /opt/docker
```

Comment:
Possibly worth an explanation and/or example of what this `target` folder is?

Reply:
Do you think? Isn't that tied into SBT / Typesafe config?
Which the link describes?

Reply:
I think now you’ve clarified that it’s a Scala app, this is fine.

(I wrote this comment first.)